packages/mongo-nav/src/on-element-click-provider/OnElementClick.spec.tsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+import { render, cleanup, act, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { nullableElement, Queries } from '@leafygreen-ui/lib/src/testHelpers';
+import MongoNav from '../MongoNav';
+import { NavElement } from '../types';
```

Comment:
Could we add a test to ensure all values are unique in the NavElement object?

Reply:
👍